code/game/objects/effects/decals/cleanable.dm

```diff
@@ -28,6 +28,7 @@
 				add_blood = bloodiness
 			bloodiness -= add_blood
 			S.bloody_shoes[blood_state] = min(MAX_SHOE_BLOODINESS, S.bloody_shoes[blood_state] + add_blood)
+			S.bloody_shoes["blood_alpha"] = 150 * (alpha/255)
```

Comment:
I am not quite sure what is going on here, why do you multiply it by 150? won't this apply to all blood and not just slime jelly?

Reply:
default is 150, I multiply it by a fraction to get the tracks to be more transparent. Maybe I should just use BLOODY_FOOTPRINT_BASE_ALPHA here